localization/english/wc_other_titles_l_english.yml

```
@@ -471,4 +471,6 @@
  
  d_neptulon_worship:0 "$NEPTULON_POSSESSIVE$ Cult"
  d_neptulon_worship_adj:0 "$NEPTULON_POSSESSIVE$"
- d_neptulon_worship_article:0 "the "
\ No newline at end of file
+ d_neptulon_worship_article:0 "the "
+
+ d_kirin_tor:0 "The Kirin Tor"
```

Seen the example above.
```suggestion
 d_kirin_tor:0 "Kirin Tor"
 d_kirin_tor_adj:0 "Kirin Tor"
 d_kirin_tor_article:0 "the "
```